src/libraries/System.Text.Encodings.Web/tools/updating-encodings.md

```diff
@@ -4,7 +4,7 @@ This folder contains tools which allow updating the Unicode data within the __Sy
 
 ### Current implementation
 
-The current version of the Unicode data checked in is __12.1.0__. The archived files can be found at https://unicode.org/Public/12.1.0/.
+The current version of the Unicode data checked in is __13.0.0__. The archived files can be found at https://unicode.org/Public/13.0.0/.
 
 ### Updating the implementation
 
```

Comment:
Down below it mentions you need `dotnet` version 3.0 or above, but somewhere else here the minimum version is now set to 3.1.

Reply:
Good eye! I updated the docs. Also had to add the `--framework netcoreapp3.1` switch manually due to a quirk from recent build system updates.